### DISCUSSION TITLE
Is there a way to show all tour steps when the page renders?

### DISCUSSION THREAD

**Discussion by dadambickford** [gilbarbara/react-joyride#952] [Q&A] [answered]
I used the custom examples [here](https://react-joyride.com/custom) as a basis for an onboarding tour recently, and got a request from our product people to see if there was a way to show all of the pulsing beacons at the start of the tour?

So in the example above we see just the first tour step, and then from that open tool tip we close it and then see the next pulsating beacon for the total of 4 steps. Our use case, if possible, would like to see all beacons pulsing on page render, allowing users to generally "start" the tour wherever they want.

I dug through the docs and didn't see anything like this, but figured I'd reach out and ask if maybe there was just another term being used for this kind of functionality that I couldn't find. And if it totally doesn't exist I'd be curious this communities thoughts on how one might accomplish it with the custom joyride approach, or maybe a future joyride feature?

Thanks in advance!

**Comment by gilbarbara** [comment, marked as answer]
Hey @dadambickford 
There isn't. This package runs sequential steps for a guided tour.

You could use the [react-floater](https://github.com/gilbarbara/react-floater) package directly and have all beacons rendered simultaneously.

**Reply by dadambickford** [reply]
thanks, I'll look into that 👍
